Expensify/App
# Add TestActions to `window` when not on production

### Details

It is difficult to simulate certain test conditions like an expired authToken or credentials now that we are not using `localStorage`. This PR adds a `TestActions` to `window` to make it easier to simulate these conditions.

### Fixed Issues
None

### Tests
1. Make sure the `.env` file has something other than `production` for the environment
2. In the JS console enter `TestActions.invalidateAuthToken()`
3. Check the network tab
4. Do something that would trigger a reauthentication and verify the next command returned a 407 jsonCode
5. In the JS console enter `TestActions.invalidateCredentials()` followed by `TestActions.invalidateAuthToken()`
6. Do something that would trigger a reauthentication (e.g. chat switch) and verify you get logged out.


### QA Steps
1. Same as Tests (But skip first step)


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
